### PR TITLE
Implements transfer opcodes

### LINKED ISSUE #48: TAX
http://obelisk.me.uk/6502/reference.html#TAX

### LINKED ISSUE #49: TXA
http://obelisk.me.uk/6502/reference.html#TXA

### LINKED ISSUE #51: TYA
http://obelisk.me.uk/6502/reference.html#TYA

### LINKED ISSUE #52: TSX
http://obelisk.me.uk/6502/reference.html#TSX

### LINKED ISSUE #53: TXS
http://obelisk.me.uk/6502/reference.html#TXS

### PR DESCRIPTION
**Pull request by jovit**
Closes #48
Closes #49
Closes #51
Closes #52
Closes #53

### CODE REVIEW

File: emulator/src/opcodes/move.py [hunk 1]
```diff
@@ -102,21 +112,34 @@ class TYA(OpCode):
...
+        cpu.negative = cpu.x & 0b10000000
+        cpu.inc_cycle()
 
 class TXS(OpCode):
     @classmethod
     def create_variations(cls):
         variations = [(0x9A, None, 2,)]
         return map(cls.create_dict_entry, variations)
-
+    def exec(cls, cpu, memory):
```
Comment (lnarita) [inline emulator/src/opcodes/move.py]: TXS does not affect flags (ref: http://obelisk.me.uk/6502/reference.html#TXS)

**Comment by jovit** [comment]
@lnarita fixed both issues, kind of odd that TXS does not affect flags

**Comment by jovit** [comment]
On a side note, are there unit tests in the project?

**Comment by lnarita** [comment]
clearly, there are none.
~bold of you to assume I had enough time to care about unit tests~

**Comment by lnarita** [comment]
> On a side note, are there unit tests in the project? Maybe we should have some hahaha

Clearly, there are none
~bold of you to assume that I had enough time and patience to care about unit tests~

**Pull request merged**